tests/test_data_passable.py

```diff
@@ -34,4 +40,65 @@ def test():
     tuner = irace(scenario, params, target_runner)
     best_conf = tuner.run()
     assert q.get() == 124
-    
\ No newline at end of file
+    
+def test_instances():
+    q = Queue()
+    scenario = dict(
+        instances = [q],
+        maxExperiments = 180,
+        debugLevel = 0,
+        parallel = 1,
+        logFile = "",
+        seed = 123
+    )
+    tuner = irace(scenario, params, target_runner2)
+    best_conf = tuner.run()
+    assert q.get() == 1335
+
+@pytest.mark.skipif(os.name == 'nt',
+                    reason="Parallel on Windows not supported")
+def test_instances2():
+    q = Queue()
+    scenario = dict(
+        instances = [q],
+        maxExperiments = 180,
+        debugLevel = 0,
+        parallel = 2,
+        logFile = "",
+        seed = 123
+    )
+    tuner = irace(scenario, params, target_runner2)
+    best_conf = tuner.run()
+    assert q.get() == 1335
+
+def test_default_serializer():
+    q = Queue()
+    scenario = dict(
+        instances = [q],
+        maxExperiments = 180,
+        debugLevel = 0,
+        parallel = 1,
+        logFile = "",
+        seed = 123,
+        instanceObjectSerializer = lambda x: 'hello world'
+    )
+    tuner = irace(scenario, params, target_runner2)
+    best_conf = tuner.run()
+    assert q.get() == 1335
+
+@pytest.mark.skipif(os.name == 'nt',
+                    reason="Parallel on Windows not supported")
+def test_default_serializer():
+    q = Queue()
+    scenario = dict(
+        instances = [q],
+        maxExperiments = 180,
+        debugLevel = 0,
+        parallel = 2,
+        logFile = "",
+        seed = 123,
+        instanceObjectSerializer = lambda x: 'hello world'
```

Comment:
What is this doing? Could you document it somewhere? 

Irace is extremely well documented and most of its documentation currently applies to iracepy. See: https://mlopez-ibanez.github.io/irace/ and https://mlopez-ibanez.github.io/irace/irace-package.pdf